addons/l10n_es_pos/models/account_move.py

```diff
@@ -10,3 +10,8 @@ def _compute_l10n_es_is_simplified(self):
         for move in self:
             if move.pos_order_ids:
                 move.l10n_es_is_simplified = move.pos_order_ids[0].is_l10n_es_simplified_invoice
+
+    def _generate_pdf_and_send_invoice(self, template, force_synchronous=True, allow_fallback_pdf=True, bypass_download=False, **kwargs):
+        if not self.company_id.l10n_es_edi_facturae_certificate_id and self.company_id.country_code == "ES":
+            kwargs = {'l10n_es_edi_facturae_checkbox_xml': False}
```

Comment:
should you not update the dict instead of overriding it ? This way if kwargs already contains some values you don't remove them